src/components/Footer.tsx

```diff
@@ -0,0 +1,256 @@
+'use client';
+
+import { useState, useMemo } from 'react';
+import Image from 'next/image';
+
+interface TeamMember {
+  name: string;
+  github: string;
+  linkedin?: string;
+  role: 'Developers' | 'UI/UX Designer' | 'Scrum Master' | 'Product Owner';
+  gender: 'male' | 'female';
+  imageName: string;
```

Comment:
```suggestion
  imageName?: string;
```
not everyone has image information, so this property should be marked as optional